cypress.json

```diff
@@ -1 +1,5 @@
-{}
+{
+  "chromeWebSecurity": true,
+  "baseUrl": "http://localhost:8000",
+  "projectId": "e7rv3w"
```

Comment:
is `projectId` a secret?

Reply:
No, it matches the test runs to the dashboard. There is not an ability to read data or access anything from this being exposed.